app/controllers/map.js

```diff
@@ -16,15 +16,15 @@ export default Controller.extend({
 
   @computed('application.model.maps', 'model.slug')
   previousNarrative(maps, currentSlug) {
-    const mapNarratives = maps.filter(map => map.hasNarrative);
+    const mapNarratives = maps.filter(map => map.isLinearNarrative);
```

Comment:
this can even be written as:
`const mapNarratives = maps.filter(({ isLinearNarrative }) => isLinearNarrative);`